service/worker/replicator/replicator.go

```diff
@@ -165,19 +168,32 @@ func (r *Replicator) createKafkaProcessors(currentClusterName string, clusterNam
 	adminClient := admin.NewRetryableClient(
 		r.clientBean.GetRemoteAdminClient(clusterName),
 		common.CreateAdminServiceRetryPolicy(),
-		common.IsWhitelistServiceTransientError,
+		isRetryableError,
+	)
+	// Create retry policy for service busy error
+	adminRetryClient := admin.NewRetryableClient(
+		adminClient,
+		common.CreateReplicationServiceBusyRetryPolicy(),
+		common.IsResourceExhausted,
 	)
 	historyClient := history.NewRetryableClient(
 		r.historyClient,
 		common.CreateHistoryServiceRetryPolicy(),
-		common.IsWhitelistServiceTransientError,
+		isRetryableError,
 	)
+	// Create retry policy for service busy error
+	historyRetryClient := history.NewRetryableClient(
+		historyClient,
+		common.CreateReplicationServiceBusyRetryPolicy(),
+		common.IsResourceExhausted,
+	)
+
```

Comment:
`IsResourceExhausted` already contains `ResourceExhausted`
plz remove this logic